### ISSUE TITLE
make nvda accessible to avaste

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7880]
I am sorry that this issue has already appeared several times.
What makes me request this feature again is that I ran the avaste with the jaws screen reader and to my amazement this worked with the software satisfactorily.
It allowed you to browse and open your home screens, and even uninstall the software.
Already when run the nvda next to the avaste and try to navigate with the arrows or tab the nvda announces unavailable.
Please please identify why the other screen reader works with the Avaste, and try to implement some functionality that will allow you to work with the software as well.
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
Install and run the last version of the avaste.
Try navigating with the arrows and tab, nvda announces legaci window and does not read any option.
Now open the uninstaller of the avaste going in the panel of contrle of windows, and likewise the nvda announces only legaci window and when navigating with the arrows or tab the nvda is muted or announces unknown.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.
The nvda should be able to read the options when navigating with tab and the arrows.
Remembering that jaws are somehow managing to read these screens.

### System configuration:
NVDA version:
> Example: next-14753

NVDA Installed or portable:
instaled
Other information:
Avaste is being offered during the software installation and cleaner.
I installed it unintentionally and I was trying to uninstall it or disable it with the use of nvda and I did not succeed.
So I was able to work with it using the other screen reader.

Windows version:
> Example: Windows 10 Version 1709 Build 16299.125

Name and version of other software in use when reproducing the issue:
avaste
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result

no
follows the nvda log when trying to navigate the avaste.
[nvda-old.log.txt](https://github.com/nvaccess/nvda/files/1591754/nvda-old.log.txt)

**Comment by Brian1Gaff** [comment]
I suspect somebody needs to talk to the Avast people on this one. I remember 
some time ago that somebody asked as you say, but the company appeared not 
to be interested due to the protective nature of the way the code works.
 One assumes Jaws has paid them some money or something for a way to get 
access. I have no proof of this, but it does seem strange otherwise.

Brian

**Comment by fernando-jose-silva** [comment]
unfortunately if this suspicion is true we are bad.

**Comment by feerrenrut** [comment]
Yes, I think the way forward with this one is to raise support requests with Avast to provide support for NVDA.

**Comment by nvdaes** [comment]
Hello: A user of a spanish mailing list for NVDA thinks that maybe possible to make Avast accessible using browse mode, like done in JAWS:

https://nvdaes.groups.io/g/lista/message/562

Thanks

**Comment by Adriani90** [comment]
The question is, whay does it work with Jaws but not with NVDA? Even with Jaws 2019 in Windows 10 it works fine. cc: @leonardder

**Comment by Carlos-EstebanM** [comment]
Update: Is possible use the last version of Avast with NVDA and the brouse mode. The installer is possible use with the revision of screen or OCR, but when Avast is installed the interface is now accessible with NVDA, because Avast have a better controls in the Window.
